sphinx/tutorial/cylc/furthertopics/clock-triggered-tasks.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+Clock Triggered Tasks
+=====================
+
+This part of the Rose user guide walks you through using clock triggered tasks.
+
+This allows you to trigger tasks based on the actual time.
+
+
+Purpose
+-------
+
+.. TODO - Insert animation / diagram.
+
+The typical use of clock triggered tasks is to perform real-time data
+retrieval.
+
+In general, most tasks in a suite do not care about the wall clock time,
+simply running once their prerequisites are met. Clock triggered tasks,
+however, wait for a particular wall clock time to be reached in addition
+to their other prerequisites.
+
+
+Clock Triggering
+----------------
+
+When clock triggering tasks we can use different
+:ref:`offsets <tutorial-iso8601-durations>` to the cycle time as follows:
+
+.. code-block:: cylc
+
+   clock-trigger = taskname(CYCLE_OFFSET)
+
+.. note::
+
+   Regardless of the offset used, the task still belongs to the cycle from
+   which the offset has been applied.
+
+
+Example
+-------
+
+Our example suite will simulate a clock chiming on the hour.
+
+Within your ``~/cylc-run`` directroy create a new directory called
+``clock-trigger``::
+
+   mkdir ~/cylc-run/clock-trigger
+   cd ~/cylc-run/clock-trigger
+
+Paste the following code into a ``suite.rc`` file:
+
+.. code-block:: cylc
+
+   [cylc]
+       UTC mode = True # Ignore DST
+
+   [scheduling]
+       initial cycle point = TODO
+       final cycle point = +P1D # Run for one day
+       [[dependencies]]
+           [[[PT1H]]]
+               graph = bell
+
+   [runtime]
+       [[root]]
+           [[[events]]]
+               mail events = failed
+       [[bell]]
+           env-script = eval $(rose task-env)
+           script = printf 'bong%.0s\n' $(seq 1 $(cylc cyclepoint --print-hour))
+
+Change the initial cycle point to 00:00 this morning (e.g. if it was
+the first of January 2000 we would write ``2000-01-01T00Z``).
+
+We now have a simple suite with a single task that prints "bong" a number
+of times equal to the (cycle point) hour.
+
+Run your suite using::
+
+   cylc run clock-trigger
+
+Stop the suite after a few cycles using the :guilabel:`stop` button in the
+``cylc gui``. Notice how the tasks run as soon as possible rather than
+waiting for the actual time to be equal to the cycle point.
+
+
+Clock Triggering Tasks
+----------------------
+
+We want our clock to only ring in real time rather than the simulated
```

Comment:
Hyphenate to 'real-time' to be (strictly correct and) consistent with the first usage of this on line 17.